custom/icds_reports/reports/daily_indicators.py

```diff
@@ -0,0 +1,85 @@
+from custom.icds_reports.models import AggAwcDailyView
+from dateutil.relativedelta import relativedelta
+from io import StringIO
+from custom.icds_reports.utils import apply_exclude
+import csv
+from datetime import date
+
+
+def _get_data_for_daily_indicators(filters, domain, show_test=False):
+    queryset = AggAwcDailyView.objects.filter(
+        **filters
+    ).values('state_name', 'num_launched_awcs', 'daily_attendance_open',
+             'total_eligible_pse', 'total_attended_pse')
+    if not show_test:
+        queryset = apply_exclude(domain, queryset)
+    return queryset
+
+
+def get_daily_indicators(domain):
+    today_date = date.today()
+    filters = {
+        'aggregation_level': 1,
+        'date': today_date
+    }
+
+    filename = 'CAS_Daily_Status_{}.csv'.format(filters['date'].strftime('%Y%m%d'))
+    filters['date'] -= relativedelta(days=1)
+    daily_yesterday = _get_data_for_daily_indicators(filters, domain)
+
+    assert daily_yesterday
```

Comment:
I don't think this should assert. I think it should return an error message, so the user can see.

Reply:
Hey, In the caller function. proper error message is passed to the user. do you see?

Reply:
I see that now, but i still think we should not be using assertions in production code. We can raise a real exception instead.

Reply:
I am sure about the use of assert in production if we handle it properly. Will read about it. But Made the change to raise a real exception. @calellowitz